docs: sanitize examples (#243)
vmware/powershell-module-for-vmware-cloud-foundation-reporting: docs/documentation/functions/Invoke-VcfHealthReport.md

## Synopsis

> Perform health checks across a VMware Cloud Foundation instance.

Reviewer: ```suggestion
Perform health checks for a VMware Cloud Foundation instance or workload domain.

```

Author: Resolved in [c2a75df](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/c2a75df236def15cdd4dc29521c92e2964de4f10)

## Syntax

### All-WorkloadDomains

```powershell

> Invoke-VcfHealthReport -sddcManagerFqdn [String] -sddcManagerUser [String] -sddcManagerPass [String] -sddcManagerLocalUser [String] -sddcManagerLocalPass [String] -reportPath [String] [-allDomains] [-failureOnly] [-darkMode] [<CommonParameters>]

Reviewer: No change needed.

Author: Resolved in [c2a75df](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/c2a75df236def15cdd4dc29521c92e2964de4f10)

```

### Specific-WorkloadDomain

```powershell

> Invoke-VcfHealthReport -sddcManagerFqdn [String] -sddcManagerUser [String] -sddcManagerPass [String] -sddcManagerLocalUser [String] -sddcManagerLocalPass [String] -reportPath [String] -workloadDomain [String] [-failureOnly] [-darkMode] [<CommonParameters>]

Reviewer: No change needed.

Author: Resolved in [c2a75df](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/c2a75df236def15cdd4dc29521c92e2964de4f10)

### Specific-WorkloadDomain

```powershell

> Invoke-VcfHealthReport -sddcManagerFqdn <String> -sddcManagerUser <String> -sddcManagerPass <String> -sddcManagerLocalUser <String> -sddcManagerLocalPass <String> -reportPath <String> -workloadDomain [String] [-failureOnly] [-darkMode] [<CommonParameters>]

Reviewer: ```suggestion
Invoke-VcfHealthReport -sddcManagerFqdn <String> -sddcManagerUser <String> -sddcManagerPass <String> -sddcManagerLocalUser <String> -sddcManagerLocalPass <String> -reportPath <String> -workloadDomain <String> [-failureOnly] [-darkMode] [<CommonParameters>]

```

Author: Resolved in [173ab0c](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/173ab0c23e803628ed90cf61642ae84325539235)

### Example 1

Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [local_username] -sddcManagerLocalPass [local_password] -reportPath [report_path] -allDomains

> Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [vm_user] -sddcManagerLocalPass [vm_user_password] -reportPath [report_path] -allDomains

Reviewer: ```suggestion
Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [vm_username] -sddcManagerLocalPass [vm_user_password] -reportPath [report_path] -allDomains

```

Author: Resolved in [173ab0c](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/173ab0c23e803628ed90cf61642ae84325539235)

```

This example runs a health check across a VMware Cloud Foundation instance.

### Example 2
Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [local_username] -sddcManagerLocalPass [local_password] -reportPath [report_path] -workloadDomain [workload_domain_name]

```powershell

> Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [vm_user] -sddcManagerLocalPass [vm_user_password] -reportPath [report_path] -workloadDomain sfo-w01

Reviewer: ```suggestion
Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [local_username] -sddcManagerLocalPass [local_password] -reportPath [report_path] -workloadDomain sfo-w01

```

Author: Resolved in [173ab0c](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/173ab0c23e803628ed90cf61642ae84325539235)

```

This example runs a health check for a specific workload domain within a VMware Cloud Foundation instance.

Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [local_username] -sddcManagerLocalPass [local_password] -reportPath [report_path] -allDomains -failureOnly

```powershell

> Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [vm_user] -sddcManagerLocalPass [vm_user_password] -reportPath [report_path] -allDomains -failureOnly

Reviewer: ```suggestion
Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [local_username] -sddcManagerLocalPass [local_password] -reportPath [report_path] -allDomains -failureOnly

```

Author: Resolved in [173ab0c](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/173ab0c23e803628ed90cf61642ae84325539235)

### Example 1

```powershell

> Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [vm_user] -sddcManagerLocalPass [vm_user_password] -reportPath [report_path] -allDomains

Reviewer: ```suggestion
Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [local_username] -sddcManagerLocalPass [local_password] -reportPath [report_path] -allDomains

```

Author: Resolved in [c2a75df](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/c2a75df236def15cdd4dc29521c92e2964de4f10)

```

This example runs a health check across a VMware Cloud Foundation instance.

### Example 2

```powershell

> Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [vm_user] -sddcManagerLocalPass [vm_user_password] -reportPath [report_path] -workloadDomain sfo-w01

Reviewer: ```suggestion
Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [local_username] -sddcManagerLocalPass [local_password] -reportPath [report_path] -workloadDomain [workload_domain_name]

```

Author: Resolved in [c2a75df](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/c2a75df236def15cdd4dc29521c92e2964de4f10)

```

This example runs a health check for a specific workload domain within a VMware Cloud Foundation instance.

### Example 3

```powershell

> Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [vm_user] -sddcManagerLocalPass [vm_user_password] -reportPath [report_path] -allDomains -failureOnly

Reviewer: ```suggestion
Invoke-VcfHealthReport -sddcManagerFqdn [sddc_manager_fqdn] -sddcManagerUser [admin_username] -sddcManagerPass [admin_password] -sddcManagerLocalUser [local_username] -sddcManagerLocalPass [local_password] -reportPath [report_path] -allDomains -failureOnly

```

Author: Resolved in [c2a75df](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/commit/c2a75df236def15cdd4dc29521c92e2964de4f10)
